packages/core/plugins/core-plugins-base-server-internal/README.md

```diff
@@ -0,0 +1,3 @@
+# @kbn/core-plugins-base-server-internal
+
+This package contains base internal types of the `plugins` domain used across other core domains.
```

Comment:
As encountered in some other places, and described [here](https://github.com/elastic/kibana/issues/134137#issuecomment-1259050057) we had a cyclic dependency between the `plugins` domain and `rendering`/`httpResources`:

`plugins` -> `CoreSetup/Start` -> `rendering` -> `plugins`.

I broke the cycle by introducing this `base` package.